src/vma/dev/qp_mgr.cpp

```diff
@@ -536,15 +554,21 @@ void qp_mgr::mlx5_send(vma_ibv_send_wr *p_send_wqe)
 	 * implementations may use move-string-buffer assembler instructions,
 	 * which do not guarantee order of copying.
 	 */
-	COPY_64B_NT(dst, m_sq_hot_wqe);
+	mlx5_bf_copy((unsigned long long *)((uintptr_t)m_sq_bf_reg + m_sq_bf_offset),
```

Comment:
@igor-ivanov 
1. Why don't you use *volatile* as in the original code? 
2. What is the reason for changing the casting from uintptr_t to unsigned long long? 

Can you restore both volatile and casting?